build/common.sh

```diff
@@ -1228,7 +1228,7 @@ function kube::release::gcs::copy_release_artifacts() {
 
   gsutil ls -lhr "${gcs_destination}" || return 1
 
-  if [[ -n "${KUBE_GCS_RELEASE_BUCKET_MIRROR:-}" ]]; then
+  if [[ "${KUBE_GCS_RELEASE_BUCKET_MIRROR:-}" != "${KUBE_GCS_RELEASE_BUCKET}" ]]; then
```

Comment:
If `KUBE_GCS_RELEASE_BUCKET_MIRROR` is empty but `KUBE_GCS_RELEASE_BUCKET` is not will this break?